astropy/visualization/mpl_normalize.py

```diff
@@ -77,6 +77,10 @@ class ImageNormalize(Normalize):
         matplotlib colormap "under" value (i.e., any finite value < 0).
         If `None`, then NaN values are not replaced.  This keyword has
         no effect if ``clip=True``.
+
+    Notes
+    -----
+    If ``vmin == vmax``, the input data will be mapped to 0.
```

Comment:
p.s. Are you sure people won't want to be able to specify their own fill value (e.g., NaN)? Or maybe it does not matter?